File: src/main/kotlin/io/swagger/client/utils/Base64Util.java
package io.swagger.client.utils;

import java.util.Base64;
import java.nio.charset.StandardCharsets;

public class Base64Util {

    public static String toBase64(String input) {
        return Base64.getEncoder().encodeToString(input.getBytes());
    }

    public static String fromBase64(String base64) {
        return new String(Base64.getDecoder().decode(base64), StandardCharsets.UTF_8);
    }
}

File: src/main/kotlin/io/swagger/client/utils/Base64Util.kt
package io.swagger.client.utils

import java.nio.charset.StandardCharsets
import java.util.*

object Base64Util {
    fun toBase64(input: String): String {
        return Base64.getEncoder().encodeToString(input.toByteArray())
    }

    fun fromBase64(base64: String?): String {
        return String(Base64.getDecoder().decode(base64), StandardCharsets.UTF_8)
    }
}